lib/ueberauth/strategy/github/oauth.ex

```diff
@@ -28,7 +28,11 @@ defmodule Ueberauth.Strategy.Github.OAuth do
   These options are only useful for usage outside the normal callback phase of Ueberauth.
   """
   def client(opts \\ []) do
-    config = Application.get_env(:ueberauth, Ueberauth.Strategy.Github.OAuth)
+    config = :ueberauth
```

Comment:
@paulanthonywilson could you please move the start of the pipe to a newline and indent, like we've done on line 36-37 below?